cli/test/smokehouse/test-definitions/offline-ready.js

```diff
@@ -37,35 +33,7 @@ const config = {
  * offline with a service worker.
  */
 const expectations = {
-  artifacts: {
-    WebAppManifest: {
-      value: {
-        icons: {
-          value: [
-            {value: {src: {value: 'http://localhost:10503/launcher-icon-0-75x.png'}}},
-            {value: {src: {value: 'http://localhost:10503/launcher-icon-1x.png'}}},
-            {value: {src: {value: 'http://localhost:10503/launcher-icon-1-5x.png'}}},
-            {value: {src: {value: 'http://localhost:10503/launcher-icon-2x.png'}}},
-            {value: {src: {value: 'http://localhost:10503/launcher-icon-3x.png'}}},
-          ],
-        },
-      },
-    },
-    InstallabilityErrors: {
-      errors: [
-        // Icon errors were consolidated in M118
-        // https://bugs.chromium.org/p/chromium/issues/detail?id=1476999
-        {
-          _minChromiumVersion: '118',
-          errorId: 'no-acceptable-icon',
-        },
-        {
-          _maxChromiumVersion: '117',
-          errorId: 'no-icon-available',
-        },
-      ],
-    },
-  },
+  artifacts: {},
```

Comment:
This test (as well as `offline-online-only`) still verify the results of several audits, but those results don't seem super meaningful in the context of a page that is offline ready (e.g. why do we need to test `image-alt` in this smoke test?).

With the removal of installability criteria, I think we can just drop all of the `offline-*` smokes. WDYT?